gui/directoryservice/migrations/0015_add_ldaps_validation.py

```diff
@@ -0,0 +1,56 @@
+from django.db import migrations, models
+
+
+def migrate_to_ldaps_ad(apps, schema_editor):
+    AD = apps.get_model(f'directoryservice.activedirectory')
+    for o in AD.objects.all():
+        if o.ad_dns_timeout == 60:
+            o.ad_dns_timeout = 10
+
+        if o.ad_timeout == 60:
+            o.ad_timeout = 10
+
+        if not o.ad_enable:
+            o.ad_ssl = 'on'
+            o.ad_ldap_sasl_wrapping = 'sign'
```

Comment:
I'm glad we talked about this! This looks good, however, I'm interested to know why we're defaulting to enabling LDAPS for AD based on the justification that it's needed for Samba AD related services. I thought that functionality was removed from freeNAS?

Reply:
FreeNAS can't be an AD DC, but it can still join Samba AD domains.

Reply:
This recent change resolves my question. Thanks!